docs/specifications/methods/undelegated-test.md

```diff
@@ -0,0 +1,56 @@
+# Undelegated tests
+
+## Objective
+
+The purpose of an undelegated test is the test a future delegation before it is
+put in production.
+
+The purpose of an undelegated test is to test a possible future delegation 
```

Comment:
This sentence is almost identical to the previous one. Remove one of them.

Reply:
Will fix.